IdentityCore/src/cache/ios/MSIDKeychainTokenCache.m

```diff
@@ -327,8 +331,86 @@ - (MSIDAccountCacheItem *)accountWithKey:(MSIDCacheKey *)key
     return accountItems;
 }
 
+#pragma mark - Metadata
+
+- (BOOL)saveAppMetadata:(MSIDAppMetadataCacheItem *)item
+                    key:(MSIDCacheKey *)key
+             serializer:(id<MSIDAppMetadataItemSerializer>)serializer
+                context:(id<MSIDRequestContext>)context
+                  error:(NSError **)error
+{
+    assert(item);
+    assert(serializer);
```

Comment:
nit: nsassert and return NO with error if !item and !serializer. assert in release code will not stop the flow.

Reply:
fixed :)